config/prod.exs

```diff
@@ -15,12 +15,37 @@ use Mix.Config
 # which you typically run after static files are built.
 config :sejfguru, SejfguruWeb.Endpoint,
   load_from_system_env: true,
-  url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/cache_manifest.json"
+  url: [host: System.get_env("BASE_URL"), port: 80],
+  cache_static_manifest: "priv/static/cache_manifest.json",
+  root: ".",
+  server: true
 
 # Do not print debug messages in production
 config :logger, level: :info
 
+config :sejfguru, SejfguruWeb.Endpoint,
+  secret_key_base: System.get_env("SECRET_KEY_BASE")
+
+config :sejfguru, Sejfguru.Repo,
+  adapter: Ecto.Adapters.Postgres,
+  username: System.get_env("DB_USERNAME"),
+  database: System.get_env("DB_NAME"),
+  password: System.get_env("DB_PASSWORD"),
+  hostname: System.get_env("DB_HOST"),
+  port: System.get_env("DB_PORT"),
+  pool_size: 10
+
+config :sejfguru, SejfguruWeb.Guardian,
+  secret_key: System.get_env("GUARDIAN_SECRET_KEY_BASE")
+
+config :ueberauth, Ueberauth,
+  providers: [
+    google: {Ueberauth.Strategy.Google, [request_path: "/auth", callback_path: "/auth-callback", default_scope: "email", hd: System.get_env("UEBERAUTH_GOOGLE_DOMAIN")]}
```

Comment:
@nekath Could you make default_scope as `email profile` so we can get rid of blank first_name etc?